### PR TITLE
Updating weblate reference codes.

### LINKED ISSUE #3929: Update i18n docs to account for Weblate / GitLab infrastructure migration
## Description

This must be done prior to updating the strings for the 0.11.0 release:

- Point to [our GitHub translations repo](https://github.com/freedomofpress/securedrop-i18n) in: [i18n_tool.py](https://github.com/freedomofpress/securedrop/blob/develop/securedrop/i18n_tool.py)
- Update soon-to-be-404ing link in the README here: https://github.com/freedomofpress/securedrop#acknowledgments
- Update i18n docs: https://github.com/freedomofpress/securedrop/blob/2455e4e50d83075fb9ce5a6945882410c3612d8d/docs/development/i18n.rst, which still refers to the GitLab workflow and .club URLs

### PR DESCRIPTION
**Pull request by ultimatecoder**
Update docs to account for [Weblate](http://weblate.securedrop.org)

## Status

Completed

## Description of Changes

Partially fixes #3929 

## Testing

* Go through all the changed documents. Verify that they are pointing to correct reference.
* URLs are not broken

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

**Comment by ultimatecoder** [comment]
@kushaldas Thanks for reviewing this. I have updated the files according to your comment. I am apologized if I have increased the troubles. Have a great day!

File: securedrop/i18n_tool.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ class I18NTool(object):
     #
     # The database of support language, indexed by the language code
     # used by weblate (i.e. whatever shows as CODE in
-    # https://weblate.securedrop.club/projects/securedrop/securedrop/CODE/
+    # https://github.com/freedomofpress/securedrop-i18n/tree/i18n/securedrop/translations
```
Comment (kushaldas) [inline securedrop/i18n_tool.py]: Just change the .club URL to the .org URL here.

**Comment by kushaldas** [comment]
Also do not say that this fixes #3929, we sill have to change other things on top of this. You can remove that line from the comment and say this is a partial fix.

**Comment by ultimatecoder** [comment]
@kushaldas I have updated the PR description and the code according to your comments. Please review them again. Thanks!

**Pull request merged**